packages/plugin/src/schema.ts

```diff
@@ -30,27 +30,25 @@ export function getSchema(options: ParserOptions, gqlConfig: GraphQLConfig): Gra
 
   // We first try to use graphql-config for loading the schema, based on the type of the file,
   // We are using the directory of the file as the key for the schema caching, to avoid reloading of the schema.
-  if (options && options.filePath && !options.skipGraphQLConfig) {
+  if (gqlConfig && options?.filePath) {
```

Comment:
Here, we don't have to check for `options.skipGraphqlConfig` since `graphqlConfig` is null this case.

Reply:
You are right, but there is a specific use case, where you install eslint and this plugin in a monorepo that has graphql-config in the root. This will force graphql-eslint to use graphql-config even if you don't want to. I added this for flexibility and for allowing to explicitly skip graphql-config.

Reply:
I don't understand :-)

In `loadGraphqlConfig`, there is a check that returns `null` if `options.skipGraphQLConfig` is true.

Then, the loaded `gqlConfig` (`null` in our case) will be passed along with `options` to `getSchema`. So here, I don't understand how we can have both `gqlConfig` not `null` and `options.skipGraphQLConfig` be `true` ?

Reply:
So there are 2 possible flows here:
1. If you have `graphql-config` and you wish to use it - `loadGraphqlConfig` will load and return the parsed graphql-config object. Then, when it's passes to `getSchema` - it knows to use it, instead of using the `option.schema` configuration. 
2. If you have `graphql-config` and don't want `graphql-eslint` to use it - you set `skipGraphQLConfig` and it will use `null` for the graphql-config object, like it was not found or not available (basically, fallbacks to 3)
3. If you have `null` in `gqlConfig` => we are going to use `@graphql-tools` directly to load the schema and the operations, instead of using graphql-config.